CMakeLists.txt

```diff
@@ -70,19 +70,13 @@ add_compile_definitions(
     MAKOU_REACTOR_VERSION_TWEAK=0
 )
 
-find_package(QT NAMES Qt6 COMPONENTS Widgets OpenGLWidgets OpenGL Svg LinguistTools REQUIRED)
-find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets OpenGLWidgets Svg OpenGL LinguistTools REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Widgets OpenGLWidgets Svg OpenGL LinguistTools Core5Compat Xml Quick REQUIRED)
 
-set(REQUIRED_FF7TK_VERSION 0.82.0)
-find_package(ff7tk ${REQUIRED_FF7TK_VERSION} REQUIRED COMPONENTS ff7tk ff7tkQtWidgets ff7tkFormats ff7tkUtils)
+set(REQUIRED_FF7TK_VERSION 0.83.0)
+find_package(ff7tk ${REQUIRED_FF7TK_VERSION} REQUIRED COMPONENTS ff7tk fftkData ff7tkQtWidgets ff7tkFormats ff7tkUtils)
 
 find_package(ZLIB REQUIRED)
 
-if(${QT_VERSION_MAJOR} EQUAL 6)
```

Comment:
There no need to remove this if, please keep it

Reply:
Or yes remove it, it is not important

Reply:
We can't build with Qt5 anymore